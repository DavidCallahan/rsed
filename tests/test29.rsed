$text = "1234567"
print substr($text,1)
print substr($text,1,3)
print substr($text,(-3))
print substr($text,(-3),1)
