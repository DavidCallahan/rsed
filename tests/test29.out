234567
234
567
5
